internal/ackhandler/ecn.go

```diff
@@ -259,6 +261,18 @@ func (e *ecnTracker) HandleNewlyAcked(packets []*packet, ect0, ect1, ecnce int64
 	return e.state == ecnStateCapable && newECNCE > 0
 }
 
+// detectMangling fails ECN validation if all testing packets are lost or CE-marked.
+func (e *ecnTracker) detectMangling() {
```

Comment:
NIT: `detectMangling` reads like it should return a bool. But I also don't have a good suggestion. The best I can think of is `failIfMangled`

Reply:
Makes sense, I'll take it.